manage.py

```diff
@@ -121,20 +124,22 @@ def pull_prod_data(default_user_password=None):
     command = "scripts/get_data.sh %s %s" % (prod_db, out_file)
     subprocess.call(shlex.split(command))
 
-    for tbl in reversed(db.metadata.sorted_tables):
-        if tbl.name not in inspect(db.engine).get_view_names():
-            db.engine.execute(tbl.delete())
-
-    db.session.execute("DELETE FROM alembic_version")
-
+    # Drop all of the existing tables before doing a pg_restore from scratch
+    db.session.execute("DROP SCHEMA public CASCADE;")
```

Comment:
This is probably a bit aggressive for local/dev databases. It _might_ be OK for us now, but it feels like we could also wipe a dev's entire database which contains other tables. Maybe. That probably shouldn't happen if they're isolating things nicely on their machine .. but.

On the other hand, I'm not sure of a nicer way to do this. We can do the same as we did before - grab the tables sqlalchemy knows about and delete those, then manually delete the alembic version table, but that's kinda meh.

Something to think about. This is fine for _me_ ... maybe worth checking with @frankieroberto though if we're going to do this (I doubt it affects him either, but I know he works on more other projects than us 🤷‍♂️ )

Reply:
Actually this doesn't even work for me: `sqlalchemy.exc.ProgrammingError: (psycopg2.ProgrammingError) must be owner of schema public`

I don't think I have a _that_ weird setup 🤔

Reply:
Weird, I also have `DATABASE_URL=postgresql://postgres@localhost:5432/rdcms` and all the tables are owned by `postgres`. 🤔

Reply:
Mmm. I've updated the schema owner to `postgres`, so that'll patch things for me. That's probably fine for now...